src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -142,8 +149,8 @@ static InitChainEntry sInitChain[] = {
 
 void EnSkb_Init(Actor* thisx, PlayState* play) {
     EnSkb* this = (EnSkb*)thisx;
-    s16 paramOffsetBody;
     s16 paramOffsetArm;
+    s16 paramOffsetBody;
```

Comment:
i know this wasnt your change, but imo theres no need to say in the temp name that it came from params. `armOffset` and `bodyOffset` seem fine